### ISSUE TITLE
Update Jellyseerr with errors

### ISSUE THREAD

**Issue by pafade89** [tteck/Proxmox#3330]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

Hi @tteck after you do some changes with Commits on Jun 24, 2024 I got the following error at the first time of starting the update script (see logs below). Second try gives the following message: Jellyseerr is already up to date.

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

```
       __     ____
      / /__  / / /_  __________  ___  __________
 __  / / _ \/ / / / / / ___/ _ \/ _ \/ ___/ ___/
/ /_/ /  __/ / / /_/ (__  )  __/  __/ /  / /
\____/\___/_/_/\__, /____/\___/\___/_/  /_/
              /____/
 |remote: Enumerating objects: 304, done.
remote: Counting objects: 100% (206/206), done.
remote: Compressing objects: 100% (67/67), done.
remote: Total 304 (delta 158), reused 166 (delta 139), pack-reused 98
Receiving objects: 100% (304/304), 212.70 KiB | 3.13 MiB/s, done.
Resolving deltas: 100% (159/159), completed with 84 local objects.
From https://github.com/Fallenbagel/jellyseerr
   96ba53fe..31ca32df  develop                -> origin/develop
 * [new branch]        fix-abort-media-availabilitysync-invalidauth -> origin/fix-abort-media-availabilitysync-invalidauth
 * [new branch]        gh-pages               -> origin/gh-pages
 + 65b7e823...85ddedbe switch-to-fetch-api    -> origin/switch-to-fetch-api  (forced update)
 * [new tag]           preview-fetch-refactor -> preview-fetch-refactor
 \
[ERROR] in line 69: exit code 0: while executing command yarn install &> /dev/null
```

**Comment by tteck** [comment]
The change was due to this [issue](https://github.com/tteck/Proxmox/issues/3254) I switched to the dev branch

I just switched back to `yarn` (main branch) with no issues. 
I see now that `yarn install` needs a few parameters `CYPRESS_INSTALL_BINARY=0 yarn install --frozen-lockfile --network-timeout 1000000`
You may have to create a new Jellyseerr LXC

**Comment by pafade89** [comment]
Also fixes #3191

**Comment by pafade89** [comment]
Hello @tteck,

today I want to install a new Jellyseerr LXC and got the following error:

`[ERROR] in line 44: exit code 0: while executing command CYPRESS_INSTALL_BINARY=0 yarn install --frozen-lockfile --network-timeout 1000000 &> /dev/null
`

When I try to update my existing LXC I got the following error:

`[ERROR] in line 69: exit code 0: while executing command CYPRESS_INSTALL_BINARY=0 yarn install --frozen-lockfile --network-timeout 1000000 &> /dev/null
`


Should I open a new issue or can you reopen this one?

Thank you in advanced
P

**Comment by pafade89** [comment]
fixed with #3904
